app/utils/analytics.js

```diff
@@ -1,19 +1,21 @@
 export function analytics(eventData) {
-  if (eventData.value) {
-    window._paq.push([
-      "trackEvent",
-      eventData.category,
-      eventData.action,
-      eventData.name,
-      eventData.value,
-    ]);
-  } else {
-    window._paq.push([
-      "trackEvent",
-      eventData.category,
-      eventData.action,
-      eventData.name,
-    ]);
+  if(window._paq) {
```

Comment:
what is `_paq`?

Reply:
It is what Matomo uses to track data.

Reply:
Gotcha. Can you just add a comment to that effect?

Reply:
Happy to.  It's been added.